src/main/java/core/basesyntax/commands/Command.java

```diff
@@ -0,0 +1,45 @@
+package core.basesyntax.commands;
+
+public abstract class Command {
+
+    private String name;
+    private String description;
+    private String argDescription;
+
+    public Command(String name, String argDescription, String description) {
+        this.name = name;
+        this.argDescription = argDescription;
+        this.description = description;
+    }
+
+    abstract void execute(String... args);
+
+    public String toString() {
+        return String.format("%7s   %-25s   %-20s", name, argDescription, description);
+    }
+
+    public static void create(String... args) {
```

Comment:
lets. make it class non-abstract and method non-static and move all the implementation here. You don't need a separate class for each command

Reply:
> lets. make it class non-abstract and method non-static and move all the implementation here. You don't need a separate class for each command.

We need abstract class because: 
1. All commands have common properties.
2. Command cant exist as-is. It is need to have some additional behavior. (like exec() implementation that is abstract method in this abstract class).
3. What if i would like to add 3 - 5 new commands? Should i write all implementations in one file? What if i'd like to add 100 new commands?

I am not sure about my static method implementation but yes we do need static methods in case of separated commands classes:
1. I don't want to create new object every time i exec some command; I want to allocate memory for this once when loading JVM.
2. Сode readability improves.
3. Easy to add new commands in our console logic.
4. API - like style. (Command.exit(); looks legit;))

Reply:
1. You are using common properties only for output information(toString() method) so let's make enum for that.
2. It appears that your commands don't have an actual state, and we may not make object out of them, so you can move execute() logic for each command to separate methods inside this class(read, info, create..).
3. If you would move all logic here, you will probably see a possibility to optimize it and move some repeatable logic to separate private methods which you can use later in your 100 commands)
4. I don't see any problem that your class would be huge, of course it's only if your methods would be of normal size and won't contain the same code. That would be more readable for me(you can't just find command by name in just one file).
5. I cannot imagine scenario where you would need 100 commands for working with file but even in that case, class still would be readable, cause in most of this commands you will use the same private methods.